docs/toc.js

```diff
@@ -18,7 +18,13 @@ module.exports = {
     ["Commands", ["commands",
       ["Status", "status.md"],
       ["Element", ["element",
-        ["Click", "click.md"]
+        ["Click", "click.md"],
+        ["Active Element", "active.md"],
+        ["Find Element", "find-element.md"],
```

Comment:
I would put the finds at the very top

Reply:
Yeah I think I'll add subcategories to elements as well. There's too many for one menu.